test/data/examples/enrollmentrequest.yaml

```
@@ -1,34 +1,14 @@
-  conditions:
-    - lastTransitionTime: "2023-10-10T17:10:32Z"
-      message: Approved by auto approver
-      reason: AutoApproved
-      status: "True"
-      type: Approved
+apiVersion: api.flightctl.io/v1alpha1  
+kind: EnrollmentRequest  
+metadata:  
+  name: 13d632d86373caaeda64da4052f957faa742fc7050026748bc79065c8819d1b0  
+spec:  
+  csr: |  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)